content/docs/kitty/building.md

```diff
@@ -6,18 +6,6 @@ draft = false
 
 # Building the Kitty system
 
-## Acquire source code
-(this takes around 1.7Gb to download)
-
-```sh
-git clone https://github.com/au-ts/lionsos.git
-cd lionsos
-git submodule update --init
-cd micropython
-git submodule update --init lib/micropython-lib
-cd ..
-```
```

Comment:
Want to keep all of this except we want to instead `cd examples/kitty` and do `make submodules`.